src/components/menu-surface/menu-surface.tsx

```diff
@@ -176,30 +164,4 @@ export class MenuSurface {
             this.dismiss.emit();
         }
     }
-
-    private ensureMenuFitsInViewPort() {
```

Comment:
> refactor(picker): move calculation of dropdown menu height from menu-surface to picker

Hmm… I haven't looked at the code yet, but I think we'll be needing this calculation in all dropdowns, not just those belonging to limel-picker. So if it can't be done in `menu-surface`, we might want to extract it to a helper or something, that can be used in limel-picker, limel-menu, etc.

Again, I haven't looked at your changes yet, I just saw the commit message and wanted to make a note of it before logging off for the day.